rust/processor/migrations/2024-03-07-224504_fungible_asset_metadata_is_token_v2/up.sql

```diff
@@ -0,0 +1,9 @@
+-- Your SQL goes here
+ALTER TABLE fungible_asset_metadata
+ADD COLUMN IF NOT EXISTS is_token_v2 BOOLEAN;
+ALTER TABLE objects
+DROP COLUMN IF EXISTS is_fungible_asset,
+DROP COLUMN IF EXISTS is_token;
+ALTER TABLE current_objects
+DROP COLUMN IF EXISTS is_fungible_asset,
+DROP COLUMN IF EXISTS is_token;
```

Comment:
do we need to remove these? Maybe we keep it?

Reply:
Up to you tho.

Reply:
idk when someone would ever use this. they can just look at the fungible and token tables. and this is gonna change once we have a fungible token standard.